Ginger/GingerCoreNET/Database/NoSqlBase/GingerCouchbase.cs

```diff
@@ -164,11 +166,11 @@ private string GetBucketName(string inputSQL)
             {
                 if (Action == ActDBValidation.eDBValidationType.UpdateDB)
                 {
-                    bucketName = inputSQL.Substring(inputSQL.ToLower().IndexOf("Update ") + 7);
+                    bucketName = inputSQL.Substring(inputSQL.IndexOf(strUpdate, StringComparison.OrdinalIgnoreCase) + 7);
```

Comment:
Proper use of having string in a constant is when we refer to
inputSQL.Substring(inputSQL.IndexOf(strUpdate, StringComparison.OrdinalIgnoreCase) + strUpdate.Length)

so please replace it as above and also for line#173.